test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -62,6 +63,11 @@ _install_tools() {
   fi
 }
 
+setup_for_tune() {
+  docker pull $DEFAULT_IMAGE_FOR_TUNE
```

Comment:
Do we need this ?

Reply:
I encountered some errors when not pulling this image. But I can have try to test it again to confirm this.

Reply:
It seems that we don't need this.